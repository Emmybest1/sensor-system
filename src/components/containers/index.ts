export * from './error-boundary/error-boundary.component';
