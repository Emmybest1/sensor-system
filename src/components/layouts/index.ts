export * from './main/main.component';
export * from './footer/footer.component';
