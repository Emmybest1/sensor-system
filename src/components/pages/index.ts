export { default as Login } from './login/login.component';
export { default as SignUp } from './signup/signup.component';
export { default as Fallback } from './fallback/fallback.component';
