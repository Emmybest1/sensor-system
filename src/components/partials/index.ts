export * from './logo/logo.component';
export * from './input/input.component';
export * from './button/button.component';
