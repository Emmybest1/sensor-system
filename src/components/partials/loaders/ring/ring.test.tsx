import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import { RingLoader } from 'components/partials';

describe('<RingLoader/>', () => {
  test('should render Component', () => {
    const { container } = render(
      <Router>
        <RingLoader isLoading={1 < 2} />
      </Router>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
