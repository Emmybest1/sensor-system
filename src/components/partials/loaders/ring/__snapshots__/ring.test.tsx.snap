// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RingLoader/> should render Component 1`] = `
<div
  class="sc-hTZgZg JXbcI"
/>
`;
